### ISSUE TITLE
source file cannot be accessed on Tdarr_Node

### ISSUE THREAD

**Issue by begunfx** [HaveAGitGat/Tdarr#997]
I know this has probably been discussed in several threads, but I can't seem to wrap my head around solving it.  I have tdarr server and internal node setup on one of my synology servers via docker compose successfully.  I can process files with no issues.  I then added another tdarr node to a different synology server.  This registers successfully, but when I try to use it to process files I can an error: "source file cannot be accessed on Tdarr_Node".  I know it's related to the path mapping between node and server but I can't seem to get the details figured out.  

Note: Both servers are Synology servers.  tdarr server is running on "MediaVault" and the tdarr_node is running on "DataVault".  What am I missing to get the tdarr server to see the tdarr_node paths?

Below is my server compose file:

version: "3.9"
services:
  tdarr:
    image: ghcr.io/haveagitgat/tdarr:latest
    container_name: tdarr
    hostname: tdarr
    security_opt:
      - no-new-privileges:true
    healthcheck:
      test: wget --no-verbose --tries=1 --spider http://localhost:8265/ || exit 1
    ports:
      - 8265:8265
      - 8266:8266
    environment:
      - TZ=America/Los_Angeles
      - PUID=1034
      - PGID=100
      - serverIP=tdarr
      - serverPort=8266
      - webUIPort=8265
      - internalNode=true
      - nodeID=InternalNode
    volumes:
      - /volume1/docker/tdarr/server:/app/server
      - /volume1/docker/tdarr/configs:/app/configs
      - /volume1/docker/tdarr/logs:/app/logs
      - /volume1/video:/video
      - /volume1/music:/music
      - /volume1/photo:/photo
      - /volume1/docker/tdarr/media:/media
      - /volume1/docker/tdarr/temp:/temp
    restart: always

Below is my tdarr node compose file:
services:
  tdarr_node:
    container_name: tdarr_node
    image: ghcr.io/haveagitgat/tdarr_node:latest
    restart: unless-stopped
    network_mode: bridge
    environment:
      - TZ=America/Los_Angeles
      - PUID=1034
      - PGID=100
      - UMASK_SET=002
      - nodeID=DataVaultNode
      - serverIP=Mediavault
      - serverPort=8266
    volumes:
      - /volume1/docker/tdarr_node/configs:/app/configs
      - /volume1/docker/tdarr_node/logs:/app/logs
      - /volume1/docker/tdarr_node/media:/media
      - /volume1/docker/tdarr_node/temp:/temp

As you can see I have the media path in both the server and node.  This is where I will be putting files to be processed.  

This is a snipped of the log with the error:
2024-05-13T12:09:20.608Z 8SrnYixpt:Node[DataVaultNode]:Worker[young-yeti]:[Step W02] [C1] Running pre-process 
2024-05-13T12:09:20.608Z 8SrnYixpt:Node[DataVaultNode]:Worker[young-yeti]:Checking files can be accessed
2024-05-13T12:09:20.608Z 8SrnYixpt:Node[DataVaultNode]:Worker[young-yeti]:[1/2] R/W [-error-]:/media/source/test/Addams Family Values 1993 1080p BluRay DD 5 1 x264-playHD.mkv:Error: ENOENT: no such file or directory, access '/media/source/test/Addams Family Values 1993 1080p BluRay DD 5 1 x264-playHD.mkv'
2024-05-13T12:09:20.609Z 8SrnYixpt:Node[DataVaultNode]:Worker[young-yeti]:[2/2] The following source file cannot be accessed on Tdarr_Node aoEVoj2lu:
2024-05-13T12:09:20.609Z
2024-05-13T12:09:20.609Z "/media/source/test/Addams Family Values 1993 1080p BluRay DD 5 1 x264-playHD.mkv"

**Please provide the following information:**
- Config files  [can be found in /app/configs/ when using Docker or in the /configs folder next to Tdarr_Updater if not using Docker]
Server config:
{
  "serverPort": "8266",
  "webUIPort": "8265",
  "serverIP": "tdarr",
  "handbrakePath": "",
  "ffmpegPath": "",
  "logLevel": "INFO",
  "mkvpropeditPath": "",
  "ccextractorPath": "",
  "openBrowser": true,
  "cronPluginUpdate": ""
}

My tdarr_node config:
{
  "serverPort": "8266",
  "webUIPort": "8265",
  "serverIP": "tdarr",
  "handbrakePath": "",
  "ffmpegPath": "",
  "logLevel": "INFO",
  "mkvpropeditPath": "",
  "ccextractorPath": "",
  "openBrowser": true,
  "cronPluginUpdate": ""
}

 - OS: Synology DSM 7.2
 - Browser: Chrome
 - Version 2

**Comment by begunfx** [comment]
Update:  Okay.  So I mounted the tdarr server (mediavault) media folders on the datavault using CIFS.  The first time I tried requeing the test file it failed, but then it seemed to work after requeing again.  Not sure why the hiccup, but that seemed to work.  So for anyone else...the missing piece for me was to make sure you create a mount point on your tdarr node server and mount the media folders you want to process, then you might need to relaunch your docker container so it can see the new mounted folders.  Otherwise the config files don't need to be touched for a docker to docker container example.
